packages/yuudachi/src/commands/moderation/reference.ts

```diff
@@ -31,7 +31,7 @@ export default class extends Command<typeof ReferenceCommand> {
 		const originalCase = await getCase(interaction.guildId, caseId);
 
 		if (!originalCase) {
-			throw new Error(i18next.t("command.mod.common.errors.no_case", { case: args.case, lng: locale }));
+			throw new Error(i18next.t("command.mod.common.errors.no_case", { case: args.case.case, lng: locale }));
```

Comment:
Case ID is already calculated above on line 20; should we just reuse that here?

Reply:
Sure, why not

Reply:
Changed in 27eb9c0